guides/releasing/Updates QA.md

```diff
@@ -0,0 +1,81 @@
+# Updates QA
+
+## Expo Go (iOS and Android)
+
+- Publish an app with the new SDK version and `fallbackToCacheTimeout: 0`. Open it in Expo Go.
+- Publish an update with changes to the UI and manifest (splash background color is easy).
+- Close and restart Expo Go. Open the project again, should see the changes to the splash screen and app UI right away (ignoring `fallbackToCacheTimeout`).
+- Turn wifi off, close and restart Expo Go, open the same project; should see the most recent update along with the "cached update" alert.
```

Comment:
Should this say to turn off all network access or just Wi-Fi?

Reply:
good point -- revised all these to say "disable / reenable network access"